### PR TITLE
Update trace explorer page

### PR DESCRIPTION
**Pull request by lizokm**

### CODE REVIEW

File: docs/product/explore/traces/index.mdx [hunk 1]
```diff
@@ -1,85 +1,93 @@
...
+description: "Learn how to use the Trace Explorer to search span data, find distributed traces, and debug performance issues across your entire application stack."
 ---
 
 <Note>
 The Traces page is currently in beta. Beta features are still in progress and may have bugs. We recognize the irony.
 </Note>
 
-The [**Traces**](https://sentry.io/orgredirect/organizations/:orgslug/traces/) page lets you search for individual spans that make up a trace, linked by a trace id. This gives you a connected view of your application from the frontend to the backend and/or between different applications and workflows. You can query and find particular distributed traces to help you debug problems or optimize hot paths in your application.
+The [**Traces**](https://sentry.io/orgredirect/organizations/:orgslug/traces/) page allows you to query individual spans within traces and analyze distributed traces across your entire application stack. When you search by spans, your queries are more targeted and more precise. This helps you pinpoint specific operations that may be causeing problems (like slow database queries, API call, and so on) across multiple traces.
```
Comment (coolguyzone) [inline docs/product/explore/traces/index.mdx:11]:
```suggestion
The [**Traces**](https://sentry.io/orgredirect/organizations/:orgslug/traces/) page allows you to query individual spans within traces and analyze distributed traces across your entire application stack. When you search by spans, your queries are more targeted and more precise. This helps you pinpoint specific operations that may be causeing problems (like slow database queries, API calls, and so on) across multiple traces.
```

File: docs/product/explore/traces/index.mdx [hunk 2]
```diff
@@ -1,85 +1,93 @@
...
 
-All your search conditions should be targetting spans.
+### Span Search Conditions
 
-## Query Results
+Each set of search conditions helps you find a specific kind of span within a trace. You can either:
 
-![Page displaying search conditions for a trace and results of that search.](./img/traces-results.png)
+- Use a single set of conditions to search across all spans.
+**Example:** Single condition: Find traces where a user with email `foo@example.com` loaded a specific frontend route.
```
Comment (coolguyzone) [inline docs/product/explore/traces/index.mdx:27]:
```suggestion
**Example:** Single condition: Find traces where a user with email `ztv@example.com` loaded a specific frontend route:
```

(changing email address to match the one in the screenshot)

File: docs/product/explore/traces/index.mdx [hunk 3]
```diff
@@ -1,85 +1,93 @@
...
+![Exampe of search with one span conditions set](./img/traces-one-span-conditions-example.png)
 
-Each row in the results represents a trace, with the first row of the table auto expanded to help interpret results while searching.
+- Use multiple sets of conditions to search for traces containing all specified span types.
+**Example:** Multiple conditions: Find traces where a user with the email `foo@example.com` loaded a page on your frontend on a specific route AND a slow database query was present.
```
Comment (coolguyzone) [inline docs/product/explore/traces/index.mdx:31]:
```suggestion
**Example:** Multiple conditions: Find traces where a user with the email `ztv@example.com` loaded a page on your frontend on a specific route AND a slow database query was present:
```

File: docs/product/explore/traces/index.mdx [hunk 4]
```diff
@@ -1,85 +1,93 @@
...
+### Search Syntax
 
-## How to Search Traces
+The Query Builder uses [Sentry's standard search syntax](/concepts/search/). Here are a few key points to keep in mind:
+
+- Use built-in [span properties](/concepts/search/searchable-properties/spans/) or custom tags.
+- Combine conditions with `AND` and `OR` (with some limitations on aggregates).
+- Use functions like `count()` for advanced queries.
+
+For a list of built-in key fields used in Sentry, check out our [Searchable Properties for spans documentation](/concepts/search/searchable-properties/spans/).
```
Comment (coolguyzone) [inline docs/product/explore/traces/index.mdx:44]:
```suggestion
For a list of built-in key fields used in Sentry, check out our [Searchable Properties for Spans documentation](/concepts/search/searchable-properties/spans/).
```

File: docs/product/explore/traces/index.mdx [hunk 5]
```diff
@@ -1,85 +1,93 @@
...
+## Search Query Results
+
+![Trace Explorer results](./img/traces-results.png)
+
+Search results are displayed as a list of traces. Each row corresponds to a single trace and has the below fields:
+
+- **Trace ID**: Links to the detailed [Trace View](/concepts/key-terms/tracing/trace-view/).
+- **Project**: The associated Sentry project.
+- **Trace Name**: Derived from the root transaction or first transaction.
+- **Matching Spans**: Number of spans matching your query versus the total spans in the trace.
```
Comment (coolguyzone) [inline docs/product/explore/traces/index.mdx:61]:
```suggestion
- **Matching Spans**: The number of spans matching your query versus the total spans in the trace.
```

File: docs/product/explore/traces/index.mdx [hunk 6]
```diff
@@ -1,85 +1,93 @@
...
+- **Timeline**: Visual representation of significant spans within the trace.
```
Comment (coolguyzone) [inline docs/product/explore/traces/index.mdx:62]:
```suggestion
- **Timeline**: A visual representation of significant spans within the trace.
```

File: docs/product/explore/traces/index.mdx [hunk 7]
```diff
@@ -1,85 +1,93 @@
...
+- **Duration**: Total time from the first to last span.
```
Comment (coolguyzone) [inline docs/product/explore/traces/index.mdx:63]:
```suggestion
- **Duration**: The total time from the first to last span.
```

File: docs/product/explore/traces/index.mdx [hunk 8]
```diff
@@ -1,85 +1,93 @@
...
+- **Timestamp**: When the trace occurred (hover for exact time).
+- **Issues**: Links to any associated error issues.
+
+### Span Details
+
+Expand a trace to see the matching spans in the below table format:
```
Comment (coolguyzone) [inline docs/product/explore/traces/index.mdx:69]:
```suggestion
Expand a trace to see the matching spans in the following table format:
```

'below table format' tripped me up a little, but might just be a me problem ¯\_(ツ)_/¯

File: docs/product/explore/traces/index.mdx [hunk 9]
```diff
@@ -1,85 +1,93 @@
...
+
+- **Span ID**: Links to the span in the Trace View.
+- **Description**: Includes project, `span.op`, and `span.description`.
+- **Timeline**: Visual representation of the span's position within the trace.
```
Comment (coolguyzone) [inline docs/product/explore/traces/index.mdx:73]:
```suggestion
- **Timeline**: A visual representation of the span's position within the trace.
```

File: docs/product/explore/traces/index.mdx [hunk 10]
```diff
@@ -1,85 +1,93 @@
...
+- **Duration**: Time taken by the individual span.
```
Comment (coolguyzone) [inline docs/product/explore/traces/index.mdx:74]:
```suggestion
- **Duration**: The time taken by the individual span.
```

**Review by coolguyzone — APPROVED** [review]
Added a few suggestions, looks good!

**Comment by lizokm** [comment]
@coolguyzone thanks for the review!

**Pull request merged**